conda/recipes/rapids-build-env/meta.yaml

```diff
@@ -125,6 +125,7 @@ requirements:
     - pytest-timeout
     - pytest-xdist
     - python
+    - python-build
```

Comment:
This should be moved to the notebook environment in my opinion. We don't need this to build RAPIDS, and it makes sense in the notebook env since its required for nvdashboard.

Reply:
done